csrank/discretechoice/feta_discrete_choice.py

```diff
@@ -0,0 +1,78 @@
+import functools
+
+import torch.nn as nn
+
+from csrank.discrete_choice_losses import CategoricalHingeLossMax
+from csrank.discretechoice.discrete_choice import SkorchDiscreteChoiceFunction
+from csrank.modules.object_mapping import DenseNeuralNetwork
+from csrank.modules.scoring.feta import FETAScoring
+
+
+class FETADiscreteChoiceFunction(SkorchDiscreteChoiceFunction):
+    """A discrete choice estimator based on the FETA-Approach.
+
+    Trains a model that first evaluates each object in contexts of limited size
+    and then aggregates these evaluations to arrive at a final
+    object-within-context evaluation.
+
+    The resulting model can then be used for context-sensitive choice.
+
+    Parameters
+    ----------
+    module : torch module (class)
+        This is the scoring module. It should be an uninstantiated
+        ``torch.nn.Module`` class that expects the number of features per
+        object as its only parameter on initialization.
+
+    choice_size : int
+        The size of the target choice set.
+
+    criterion : torch criterion (class)
+        The criterion that is used to evaluate and optimize the module.
+
```

Comment:
Some parameters are missing here.

Reply:
Yes, the documentation was out of date. Thanks for noticing. Changed.